cmd/tempo/app/modules.go

```diff
@@ -274,3 +276,13 @@ func (t *App) setupModuleManager() error {
 func queryEndpoint(cfg *Config) string {
 	return cfg.HTTPAPIPrefix + "/api/traces/{traceID}"
 }
+
```

Comment:
Let me nit you just a bit more :)!

let's make a single method with some really basic tests:

```
func  applyHTTPPrefix(cfg, path) {
}
```

Also let's use `path.Join` instead of string concat.

Thanks for the work, we're almost there!

Reply:
Done! I didn't add a unit test for `addHTTPAPIPrefix` because the function is really small and not very error-prone (feel free to disagree on this!).
If we change this logic at some point, the e2e tests should break.

I did notice whoever that the e2e tests don't configure a prefix... What do you think, should I adapt one of the configs to use a prefix?